.github/renovate.json5

```diff
@@ -238,6 +241,20 @@
       ],
       "allowedVersions": ">1.6"
     },
+    {
+      "groupName": "spire-images",
+      "matchFiles": [
+        "install/kubernetes/cilium/values.yaml.tmpl"
+      ],
+      "matchPackageNames": [
+        "ghcr.io/spiffe/spire-agent",
+        "ghcr.io/spiffe/spire-server"
+      ],
+      "matchBaseBranches": [
+        "v1.15"
+      ],
+      "allowedVersions": "<1.9"
+    },
```

Comment:
Where did this came from? :thinking:

Reply:
this was added before, however, the above block will not have any impact after https://github.com/cilium/cilium/pull/27621

Reply:
I copied this from the 1.14 version and updated the allowed versions to match the current chart.

@sayboras are you saying I should drop this piece? Is that because we made SPIRE updates manual, renovate bot doesn't handle that for now?

Reply:
yes, the update is done manually currently, till we sort out the post-update step in rennovate.

Reply:
Filed https://github.com/cilium/cilium/issues/29839 to follow up.